src/app/movies/movies.component.ts

```diff
@@ -25,12 +27,20 @@ export class MoviesComponent {
     this.movieFacade.addMovie(movieInfo);
   }
 
+  sanitizeSrc(movie: Movie): SafeUrl {
+    return this.sanitizer.bypassSecurityTrustResourceUrl(movie.src as string);
+  }
+
   parseUserInput(movieInput: string): Movie {
     const videoMetadata = getVideoId(movieInput);
     return {
       type: videoMetadata.service,
-      link: movieInput,
       id: videoMetadata.id,
+      title: '',
+      publishedAt: Date.now(),
+      likeCount: '',
```

Comment:
po to dałes te opcjonalne properties w interfejsie żeby tu nie wstawiać pusych stringów niepotrzebnych do niczego ;)